testsuite/python/lb_interpolation.py

```diff
@@ -23,11 +23,11 @@
 import espressomd.shapes
 import espressomd.lb
 
-AGRID = 1.5
+AGRID = 1.5 
```

Comment:
there is an extra whitespace
```suggestion
AGRID = 1.5
```